kalite/distributed/templates/distributed/base.html

```diff
@@ -277,7 +277,7 @@
                               </a>
                               <ul class="dropdown-menu" role="menu">
                                 <li role="presentation student-only">
-                                  <span id="points student-only">
```

Comment:
This error is an oversight from a PR that I made (#3127). The intention was that `student-only` be a class. Can you change this span to have the class `student-only`, and fix the role of the `li` element on line 278? It should also have class `student-only` and the role should just be `presentation`.